integration/helpers/loadbalancer/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 
 	addr := "0.0.0.0:443"
 	server := &http.Server{Addr: addr, Handler: proxy}
-	tlsConfig := &tls.Config{
+	tlsConfig := &tls.Config{MinVersion: SSL.VersionTLS13,
```

Comment:
Similar to above: what is unique about this that it gets version 13 instead of 12?
```suggestion
	tlsConfig := &tls.Config{
	  MinVersion: SSL.VersionTLS13,
```